### ISSUE TITLE
Accessibility Fix for Toggle and Tabs - Request and Basic Fix

### ISSUE THREAD

**Issue by KnickelsTheEkidna** [soflyy/oxygen-bugs-and-features#2271]
**Describe the feature you'd like to see included in Oxygen.**

Currently in base Oxygen the toggle and tab feature does not have tabindex functionality, therefore hinders users that need accessibility help using the Keyboard Only

**What are the use cases for this feature?**

Accessibility 

**The fix I came up with**

<script>
	jQuery(".oxy-toggle").attr("tabindex","0");
	jQuery(".oxy-toggle").keyup(function(event) {
     var keycode = (event.keyCode ? event.keyCode : event.which);
    if(keycode == '13'){
    
		jQuery(document.activeElement).addClass("toggle-3947-expanded");
  
		
		
		if(jQuery(":focus div").hasClass("oxy-eci-collapsed")){
		jQuery(':focus .oxy-expand-collapse-icon').removeClass("oxy-eci-collapsed");
		} else{
			
			jQuery(':focus .oxy-expand-collapse-icon').addClass("oxy-eci-collapsed");
		}
		
		

		jQuery(document.activeElement).next('div').toggle();
  
    }
});
</script>

<script>
	jQuery(".oxy-tab").attr("tabindex","0");
	jQuery(".oxy-tab").keyup(function(event) {
     var keycode = (event.keyCode ? event.keyCode : event.which);
    if(keycode == '13'){
    
		
		var currentTabber = jQuery(":focus").parent('div').attr("data-oxy-tabs-active-tab-class");
		var currentTabberContents = "#" + jQuery(":focus").parent('div').attr("data-oxy-tabs-contents-wrapper"); 	
		var finder = jQuery(currentTabberContents).index();
		var finderLocation = jQuery(":focus").index();
		
		
		/*console.log(currentTabber);
		console.log(currentTabberContents);
		console.log(finder);
		console.log(finderLocation);*/
		
		if(jQuery(':focus').hasClass(currentTabber)){
			return;
		} else {
			
			jQuery('.oxy-tab').removeClass(currentTabber);
			jQuery(':focus').addClass(currentTabber);
			jQuery(currentTabberContents).children('.oxy-tab-content').addClass('oxy-tabs-contents-content-hidden');
			jQuery(currentTabberContents).children('.oxy-tab-content').eq(finderLocation).removeClass('oxy-tabs-contents-content-hidden');
			
			
		}  
    }
});
</script>

**Comment by c6410** [comment]
+1

**Comment by gitsimhub** [comment]
+1
